ru/mail/chrome/messenger/preferences/connection.dtd

```diff
@@ -34,4 +34,4 @@
 <!ENTITY  noproxy.accesskey             "N">
 <!ENTITY  noproxyExplain.label          "Пример: .mozilla-russia.org, .net.nz, 192.168.1.0/24">
 <!ENTITY  shareproxy.label              "Использовать этот прокси-сервер для всех протоколов">
-<!ENTITY  shareproxy.accesskey          "-">
+<!ENTITY  shareproxy.accesskey          "x">
```

Comment:
OK, sorry. I have reverted this hotkey change. Should I commit the fixed file again?